### ISSUE TITLE
breaking changes in npm version 1.3.2 ?

### ISSUE THREAD

**Issue by makc** [regl-project/regl#486]
getting this with 1.3.2 today, the same code works with 1.3.1:
![screen shot 2018-05-16 at 17 03 49](https://user-images.githubusercontent.com/242577/40121943-5766879c-592b-11e8-8230-930d6a3def58.png)

**Comment by rreusser** [comment]
Can you give a bit of code that causes it or an unminified stack trace (you can get that by requiring `regl/regl` instead of just `regl`)?

**Comment by makc** [comment]
something like this:

```javascript
var f = regl({
      frag: '....',
      vert: '....',
      attributes: {
        position: mesh.positions
      },
      elements: mesh.cells,
      uniforms: {
        ....,
        normalMap: normalMap,
        textureMap: textureMap,
      }
    });

and then
regl.frame(function () {
        regl.poll();
        regl.clear({
          color: [0, 0, 0, 255],
          depth: 1
        });
        f();
```

**Comment by rreusser** [comment]
I'm able to reproduce the error with this code:

```javascript
require('regl')({
  onDone: function (err, regl) {
    if (err) throw err;

    const mesh = require('bunny');

    const draw = regl({
      vert: `
        precision mediump float;
        attribute vec3 position;
        uniform mat4 projection, view;
        void main () {
          gl_Position = projection * view * vec4(position, 1);
        }`,
      frag: `
        precision mediump float;
        void main () {
          gl_FragColor = vec4(1);
        }`,
      attributes: {position: mesh.positions},
      elements: mesh.cells,
      count: mesh.cells.length * 3 
    }); 

    const camera = require('regl-camera')(regl);

    regl.frame(({tick}) => {
      regl.clear({color: [0, 0, 0, 1]});
      camera(() => {
        draw();
      }); 
    }); 
  }
});
```

But if I switch to `require('reg/regl')`, the error goes away. Hmm… shouldn't that be equivalent to require it that way, unless there's a difference in checked/unchecked?

**Comment by dy** [comment]
Should be fixed regl@1.3.3.
